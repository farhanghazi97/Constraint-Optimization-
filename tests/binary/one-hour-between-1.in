meeting, meeting1
meeting, meeting2
constraint, meeting1 one-hour-between meeting2
domain, meeting2, 2pm, hard
domain, meeting2, thu, hard
domain, meeting1, thu 2pm-thu 4pm, hard
