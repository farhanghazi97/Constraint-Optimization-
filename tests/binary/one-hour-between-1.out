meeting1:thu 4pm
meeting2:thu 2pm
cost:0
